docs/reference/generated/autocomplete-root.json

```diff
@@ -0,0 +1,141 @@
+      "type": "string | number | string[]",
+      "description": "The input value of the autocomplete. Use when controlled.",
+      "detailedType": "string | number | string[] | undefined"
+    },
+    "onValueChange": {
+      "type": "((value: string, eventDetails: Autocomplete.Root.ChangeEventDetails) => void)",
+      "description": "Callback fired when the input value of the autocomplete changes.",
+      "detailedType": "| ((\n    value: string,\n    eventDetails: Autocomplete.Root.ChangeEventDetails,\n  ) => void)\n| undefined"
+    },
+    "defaultOpen": {
+      "type": "boolean",
+      "default": "false",
+      "description": "Whether the popup is initially open.\n\nTo render a controlled popup, use the `open` prop instead.",
+      "detailedType": "boolean | undefined"
+    },
+    "open": {
+      "type": "boolean",
+      "description": "Whether the popup is currently open.",
+      "detailedType": "boolean | undefined"
+    },
+    "onOpenChange": {
+      "type": "((open: boolean, eventDetails: Combobox.RootInternal.ChangeEventDetails) => void)",
+      "description": "Event handler called when the popup is opened or closed.",
+      "detailedType": "| ((\n    open: boolean,\n    eventDetails: Combobox.RootInternal.ChangeEventDetails,\n  ) => void)\n| undefined"
+    },
+    "actionsRef": {
+      "type": "RefObject<Combobox.RootInternal.Actions>",
+      "description": "A ref to imperative actions.\n- `unmount`: When specified, the autocomplete will not be unmounted when closed.\nInstead, the `unmount` function must be called to unmount the autocomplete manually.\nUseful when the autocomplete's animation is controlled by an external library.",
+      "detailedType": "| React.RefObject<Combobox.RootInternal.Actions>\n| undefined"
+    },
+    "autoHighlight": {
+      "type": "boolean",
+      "default": "false",
+      "description": "Whether to automatically highlight the first item when the popup opens.",
+      "detailedType": "boolean | undefined"
+    },
+    "cols": {
+      "type": "number",
+      "default": "1",
+      "description": "The number of columns the items are rendered in grid layout.",
+      "detailedType": "number | undefined"
+    },
+    "filter": {
+      "type": "((value: Value, query: string, itemToLabel: ((value: Value) => string) | undefined) => boolean) | null",
+      "description": "Filter function used to match items vs input query.\nThe `itemToLabel` function is provided to help convert items to strings for comparison.",
+      "detailedType": "| ((\n    value: Value,\n    query: string,\n    itemToLabel: ((value: Value) => string) | undefined,\n  ) => boolean)\n| null\n| undefined"
+    },
+    "itemToValue": {
+      "type": "((value: Value) => string)",
+      "description": "Converts an item's value to its string representation for display in the input.",
+      "detailedType": "((value: Value) => string) | undefined"
+    },
+    "items": {
+      "type": "Group<Value>[] | Value[]",
```

Comment:
This shows as `any[] | Group[] | undefined` in the docs. Is this the stale cache issue again?